.github/workflows/release-plz.yml

```
@@ -28,7 +28,7 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
 
       - name: Run release-plz
-        uses: MarcoIeni/release-plz-action@f0fdffff1ff475195a1638e487cf93719e065a8e # v0.5
+        uses: MarcoIeni/release-plz-action@2d634174257b7f46339e7533865a910743a0c5c9 # v0.5

```

```suggestion
        uses: release-plz/action@2d634174257b7f46339e7533865a910743a0c5c9 # v0.5

```
Update the action org and name as it has been moved.